templates/vsphere/template-variables.rc

```diff
@@ -0,0 +1,55 @@
+export KUBERNETES_VERSION=""                    # K8s Version of the cluster, e.g. "v1.30.0"
+
+export PROXY_DISABLE=""                         # Set to "#" to disable, or "" to enable the proxy configuration
+                                                # If enabled, make sure to configure the proxy in the following section.
+
+if [ "$PROXY_DISABLE" == "#" ]; then
+  export HTTP_PROXY=""
+  export HTTPS_PROXY=""
+  export NO_PROXY=""
+else
+  export HTTP_PROXY=http://squid.internal:3128
+  export HTTPS_PROXY=http://squid.internal:3128
+  export NO_PROXY=10.0.0.0/8,192.168.0.0/16,127.0.0.1,localhost,0.0.0.0
+fi
+
+export CONTROL_PLANE_MACHINE_COUNT=1
+export WORKER_MACHINE_COUNT=1
+
+export CONTROL_PLANE_MACHINE_CPU_COUNT=4
+export CONTROL_PLANE_MACHINE_MEMORY_MB=4096
+export CONTROL_PLANE_MACHINE_DISK_GB=25
+
+export WORKER_MACHINE_CPU_COUNT=2
+export WORKER_MACHINE_MEMORY_MB=2048
+export WORKER_MACHINE_DISK_GB=25
+
+# Source: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/docs/getting_started.md
+
+export VSPHERE_USERNAME=""                      # The username used to access the remote vSphere endpoint
+export VSPHERE_PASSWORD=""                      # The password used to access the remote vSphere endpoint
+
+export VSPHERE_SERVER=""                        # The vCenter server IP or FQDN, e.g. "10.0.0.1"
```

Comment:
for AWS you don't set defaults. Please either update the AWS env (in another PR) or this rc file
https://github.com/canonical/cluster-api-k8s/blob/main/templates/aws/template-variables.rc

Reply:
@bschimke95 Not sure what you mean here

Reply:
You don't set an empty string in the AWS template e.g. `export AWS_PUBLIC_IP=`